### ISSUE TITLE
2.9.2 doesn't build on Debian Buster anymore

### ISSUE THREAD

**Issue by steffen-AEI** [git-lfs/git-lfs#3954]
**Describe the bug**
While release 2.9.1 happily built for Debian Buster, 2.9.2 apparently needs something that's only available in sid/testing (Bullseye).
debian/* files haven't changed for a long time.

**To Reproduce**
1. From the tarball, create a Debian source package.
2. Inject into Buster pbuilder.
3. FTBFS:
```
dh_auto_build: cd obj-x86_64-linux-gnu && go install -gcflags=all=\"-trimpath=/build/git-lfs-2.9.2/obj-x86_64-linux-gnu/src\" -asmflags=all=\"-trimpath=/build/git-lfs-2.9.2/obj-x86_64-linux-gnu/src\" -v -p 1 github.com/git-lfs/git-lfs github.com/git-lfs/git-lfs/commands github.com/git-lfs/git-lfs/config github.com/git-lfs/git-lfs/creds github.com/git-lfs/git-lf
s/docs/man github.com/git-lfs/git-lfs/errors github.com/git-lfs/git-lfs/filepathfilter github.com/git-lfs/git-lfs/fs github.com/git-lfs/git-lfs/git github.com/git-lfs/git-lfs/git/gitattr github.com/git-lfs/git-lfs/git/githistory github.com/git-lfs/git-lfs/lfs github.com/git-lfs/git-lfs/lfsapi github.com/git-lfs/git-lfs/lfshttp github.com/git-lfs/git-lfs/lfshttp
/standalone github.com/git-lfs/git-lfs/locking github.com/git-lfs/git-lfs/script github.com/git-lfs/git-lfs/subprocess github.com/git-lfs/git-lfs/t/cmd github.com/git-lfs/git-lfs/t/cmd/util github.com/git-lfs/git-lfs/tasklog github.com/git-lfs/git-lfs/tools github.com/git-lfs/git-lfs/tools/humanize github.com/git-lfs/git-lfs/tools/kv github.com/git-lfs/git-lfs/
tq returned exit code 2                                                                                                                                                                                                                                                                                                                                                    
make[1]: *** [debian/rules:36: override_dh_auto_build] Error 2                                                                                                                                                                                                                                                                                                             
```

**Expected behavior**
2.9.1 built for Buster, while 2.9.2 only builds for Bullseye (and sid)

**System environment**
Debian Buster, pbuilder following the suggested setup (e.g. Krafft book)

**Output of `git lfs env`**
N/A

**Additional context**
debian/control still lists `Build-Depends: [...] golang-go:native (>= 1.3.0) [...]` but the required version of Go appears to have increased several times in the past. Which version is actually required? (I'd like to have a backport of git-lfs to Stretch as well, btw.)

**Comment by bk2204** [comment]
Hey,

The required version is 1.12.  We officially support the latest version of Go (1.13) as mentioned in the README, but it is known that things build on 1.12.  We needed to bump the required version in order to get Git LFS to support newer versions of Git (in #3940).

The only issue with bumping the version becomes that we don't actually use the Debian Go packages for building our own Debian Git LFS packages, so bumping the dependency in the `Build-Depends` line will actually prevent our packages from building.

If someone would like to provide a patch to make things work with older versions that doesn't regress the fix in #3940, we're happy to accept such a patch, although of course we still only officially support the latest Go version.  We can include that patch in 2.10, which will likely be released in early January.

**Comment by bk2204** [comment]
I've updated the Debian package in #3959 so that it contains the correct dependencies.  Since it was intentional to bump the required version and nobody's stepped up to provide a better patch that doesn't require a version bump, I'm going to close this.

We do provide packages via packagecloud.io that offer binaries built for stretch and buster for those that want them.
